backend/django/core/views/api_annotate.py

```diff
@@ -33,6 +36,8 @@
 from core.utils.utils_model import check_and_trigger_model
 from core.utils.utils_redis import redis_serialize_data, redis_serialize_set
 
+model = SentenceTransformer("multi-qa-MiniLM-L6-cos-v1")
```

Comment:
Definitely need to change this to a more informative name. Between Django and SMART, model is a loaded word. :)